pkg/certificates/config.go

```diff
@@ -111,7 +111,7 @@ func (c CertServerConfig) Run(ctx context.Context, srv *server.Server) (err erro
 	}
 
 	certSrv := NewServer(
-		zap.L(),
+		zap.L(), // TODO: pass this in from somewhere
```

Comment:
Is this a todo for later?

Reply:
Yes, this would require some better handling since we should get rid of server.Service things altogether to clarify the code a bit. This is the last place that is still using it.